### PR TITLE
[FIX] Display user status along with icon

### LINKED ISSUE #16868: Display user status along with icon
### Description:
Currently in the mobile view of the user profile, one can only see the icon regarding the user status. It would be better if the mobile site mirrored the desktop site, in which both the icon as well as the description are shown since the icon alone wouldn’t be recognized by all. 

### Steps to reproduce:

1. Click on the user icon in any chat window.
2. You can see their status below their username.

### Expected behavior:

 Icon along with description.

### Actual behavior:

rocketchat

### Server Setup Information:

- Version of Rocket.Chat Server: 3.1.0-develop 
- Operating System: Linux
- NodeJS Version: 12.16.1 - x64
- MongoDB Version: 4.0.6

### PR DESCRIPTION
**Pull request by Nikhil713**
Closes #16868

Updated code shows the corresponding user-status descriptions.

Output: 
user-status

### CODE REVIEW

**Comment by Nikhil713** [comment]
@kb0304 @gabriellsh Could you review this?

File: app/theme/client/imports/components/header.css [hunk 1]
```diff
@@ -413,7 +413,7 @@
 @media (width <= 500px) {
 	.rc-header {
 		&__visual-status {
-			display: none;
+			font-size: 100%;
```
Comment (gabriellsh) [inline app/theme/client/imports/components/header.css]: What effect does `font-size: 100%` has here? Is there any specific need for it?

Comment (Nikhil713) [reply app/theme/client/imports/components/header.css]: Now I see that `font-size: 100%;` is not required. I have removed the unnecessary code. Is it fine now?

**Comment by gabriellsh** [comment]
Could you please merge with develop?

**Comment by Nikhil713** [comment]
@gabriellsh Done.

**Pull request merged**